src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -397,39 +421,39 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock \
                        eclipse/che:<version> [COMMAND]
 ----
 
-To activate jpda suspend mode for debugging Che server initialization, in the `che.env`:
+To activate jpda suspend mode for debugging Che server initialization, in the `che.env` file:
 
 ----
 CHE_DEBUG_SUSPEND=true
 ----
 
-To change che debug port, in the `che.env`:
+To change the Che debug port, in the `che.env` file:
 
 ----
 CHE_DEBUG_PORT=8000
 ----
 
 [id="production-mode"]
-== Production Mode
+== Production mode
 
-You can also build own `INIT` and `SERVER` images to have custom configuration and binaries. To do so, clone https://github.com/eclipse/che[Che repo] and copy `dockerfiles` dir to the root of your custom assembly. If your custom Che server does not need any custom configuration, you may proceed to building Che server image by executing `dockerfiles/build.sh`. Once done, tag the resulted image the way you need. If your custom Che server requires custom configuration, and you want to let users override them in `che.env`, you will need to build own init image with a custom https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env] file.
+You can also build your own `INIT` and `SERVER` images to have custom configuration and binaries. For this, clone the https://github.com/eclipse/che[Che repo] and copy the `dockerfiles` directory to the root of your custom assembly. If your custom Che server does not need any custom configuration, build the Che server image by executing the `dockerfiles/build.sh` command. Once done, tag the resulted image as needed. If your custom Che server requires custom configuration and you want to let users override them in the `che.env`file, build your own `INIT` image with a custom https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env[che.env] file.
```

Comment:
* no 'also'
* repo -> repository
* no 'if'
* missing space: `che.env`file
* split the para into a numbered procedure list